adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -873,6 +873,54 @@ public void testVerifyManifestPermissionMissingMultiPermissions() throws Interru
         }
     }
 
+    public void testSilentRequestMissingPermissionHandlingForAndroid22andBelow() throws InterruptedException, PackageManager.NameNotFoundException {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
```

Comment:
I've tried. But I don't think we could do a mock on the either PackageManager.PackageInfo.versionCode or Build.VERSION.SDK_INT. Because they are both int variables not methods, and they don't have getters for us to pass the mock.